backend-shared/config/db/gitopsengineinstance_test.go

```diff
@@ -1,218 +1,65 @@
-package db
+package db_test
 
 import (
 	"context"
-	"strings"
-	"testing"
 
-	"github.com/stretchr/testify/assert"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	db "github.com/redhat-appstudio/managed-gitops/backend-shared/config/db"
 )
 
-func TestGetGitopsEngineInstanceById(t *testing.T) {
-	SetupforTestingDB(t)
-	defer TestTeardown(t)
-	dbq, err := NewUnsafePostgresDBQueries(true, true)
-	if !assert.NoError(t, err) {
-		return
-	}
-	defer dbq.CloseDatabase()
-
-	ctx := context.Background()
-
-	clusterCredentials := ClusterCredentials{
-		Clustercredentials_cred_id:  "test-cluster-creds-test-1",
-		Host:                        "host",
-		Kube_config:                 "kube-config",
-		Kube_config_context:         "kube-config-context",
-		Serviceaccount_bearer_token: "serviceaccount_bearer_token",
-		Serviceaccount_ns:           "Serviceaccount_ns",
-	}
-
-	gitopsEngineCluster := GitopsEngineCluster{
-		Gitopsenginecluster_id: "test-fake-cluster-1",
-		Clustercredentials_id:  clusterCredentials.Clustercredentials_cred_id,
-	}
-
-	gitopsEngineInstanceput := GitopsEngineInstance{
-		Gitopsengineinstance_id: "test-fake-engine-instance-id",
-		Namespace_name:          "test-fake-namespace",
-		Namespace_uid:           "test-fake-namespace-1",
-		EngineCluster_id:        gitopsEngineCluster.Gitopsenginecluster_id,
-	}
-	err = dbq.CreateClusterCredentials(ctx, &clusterCredentials)
-	if !assert.NoError(t, err) {
-		return
-	}
-
-	err = dbq.CreateGitopsEngineCluster(ctx, &gitopsEngineCluster)
-	if !assert.NoError(t, err) {
-		return
-	}
-
-	err = dbq.CreateGitopsEngineInstance(ctx, &gitopsEngineInstanceput)
-	if !assert.NoError(t, err) {
-		return
-	}
-	gitopsEngineInstanceget := GitopsEngineInstance{
-		Gitopsengineinstance_id: "test-fake-engine-instance-id",
-		Namespace_name:          "test-fake-namespace",
-		Namespace_uid:           "test-fake-namespace-1",
-		EngineCluster_id:        gitopsEngineCluster.Gitopsenginecluster_id,
-	}
-
-	err = dbq.GetGitopsEngineInstanceById(ctx, &gitopsEngineInstanceget)
-	if !assert.NoError(t, err) {
-		return
-	}
-	assert.Equal(t, gitopsEngineInstanceput, gitopsEngineInstanceget)
-
-	//check for inexistent primary key
-
-	gitopsEngineInstanceNotExist := GitopsEngineInstance{
```

Comment:
As mentioned above, I recommend preserving this 'inexistent primary key' test and moving it over to the Ginkgo version.